matRad_fluenceOptimization.m

```diff
@@ -85,7 +81,13 @@
                                         
                 error([cst{i,6}(j).type ' objective not supported ' ...
                     'during biological optimization for carbon ions']);
-           
+            end
+            if strcmp(cst{i,6}(j).type,'square underdosing') || ...
```

Comment:
Why not leave this check for the type out? That way we would also directly check for mean& eud objectives and constraints!